packages/bindings/src/serialport_unix.cpp

```diff
@@ -76,7 +76,7 @@ int ToDataBitsConstant(int dataBits) {
   return -1;
 }
 
-void EIO_Open(napi_env env, void* req) {
+void EIO_Open(void* req) {

```

Comment:
env was never used, and I believe it should not be used within the Execute method of the AsyncWorker